### PR TITLE
add some test data and add unique schema

### PR DESCRIPTION
**Pull request by TheSpaceCuber**
* customers and employees should have unique names, emails, hp_nos etc
* test data still incomplete
* moved name in customers table to the top

### CODE REVIEW

**Comment by TheSpaceCuber** [comment]
Customers, Employees, Course_packages and Rooms updated

File: project.sql [hunk 1]
```diff
@@ -30,10 +30,10 @@ CREATE TABLE Rooms (
 -- DONE
 CREATE TABLE Employees (
     eid                             INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
-    name                            VARCHAR NOT NULL,
-    phone                           INT NOT NULL,
+    name                            VARCHAR NOT NULL UNIQUE,
```
Comment (Ben-Hanan) [inline project.sql]: I dont think name should be unique!

File: project.sql [hunk 2]
```diff
@@ -99,10 +99,10 @@ CREATE TABLE Courses (
 -- DONE
 CREATE TABLE Customers (
     cust_id                         INT UNIQUE GENERATED ALWAYS AS IDENTITY,
-    phone                           INT NOT NULL,
+    name                            VARCHAR NOT NULL UNIQUE,
```
Comment (Ben-Hanan) [inline project.sql]: Same here

**Review by Ben-Hanan — APPROVED** [review]
Not sure about the names and other attributes being tagged as `UNIQUE` but LGTM

**Pull request merged**